scripts/build.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+
+# run this script from the project root: ./scripts/builds.sh
+# It will build the entire project for production
+
+set -e
+mkdir -p build
+cmake -S . -B build
```

Comment:
The actual "production" needs the `-DSTATIC_COMPILE=TRUE` flag here I think, so the resulting binary can simply be copied to the deployment server

Reply:
OK, that sounds good, just made the change